libyul/optimiser/UnusedStoreEliminator.h

```diff
@@ -88,12 +87,19 @@ class UnusedStoreEliminator: public UnusedStoreBase
 		std::optional<YulName> start;
 		/// Length of affected area, unknown if not provided.
 		/// Unused for storage.
-		std::optional<YulName> length;
+		std::optional<OperationLength> length;
 	};
 
 private:
-	std::set<Statement const*>& activeMemoryStores() { return m_activeStores["m"_yulname]; }
-	std::set<Statement const*>& activeStorageStores() { return m_activeStores["s"_yulname]; }
+	std::set<Statement const*>& activeMemoryStores() { return m_activeStores[UnusedStoreEliminatorKey::Memory]; }
+	std::set<Statement const*>& activeStorageStores() { return m_activeStores[UnusedStoreEliminatorKey::Storage]; }
+	std::optional<u256> lengthValue(OperationLength const& _length) const
```

Comment:
I'd give it a name more resembling the knowledge base function it replaces. `lengthValue()` sounds a bit plain and it might be too easy to not realize it's not doing something much simpler.

```suggestion

	std::optional<u256> lengthValueIfKnownConstant(OperationLength const& _length) const
```

Reply:
I think it's also fine like it is, the implementation is concise and right there - although I am not opposed to the longer name, either.

Reply:
When you read the code, you see the name, not the implementation and it should reflect important things about the function. IMO in this case it does not.

Reply:
We should also stop renaming and refactoring random stuff in PRs just because they happen to appear in the diff. It's not a new name introduced in the PR - if you're bothered by it, feel free to create a PR to rename it (even though I'd say the current name is significantly better in this particular case), but it has nothing to do with what this PR does.

Reply:
oh damn, it actually is new :-D sorry, I didn't see it properly on my phone :-)

Reply:
I'd still keep the name. The return type makes things pretty clear anyways